.github/workflows/primer_comment.yaml

```diff
@@ -98,9 +98,17 @@ jobs:
                archive_format: "zip",
             });
             fs.writeFileSync("primer_pr_number.zip", Buffer.from(downloadThree.data));
+      - run: sudo apt install -y jq
```

Comment:
`jq` is already installed on Github action runners.

Reply:
my bad. will remove